### PR TITLE
Refactor outfit skillchips

### PR DESCRIPTION
**Pull request by coiax**
- Skillchips now are on /datum/outfit, rather than /datum/outfit/job,
  since we may want other roles to start with skillchips in the future.
- Skillchips are now a list of types to install, rather than a single
  one.
- Traits for the deathsquad are now listed just below the deathsquad
  initial antag datum definition.

### CODE REVIEW

File: code/modules/jobs/job_types/janitor.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	ears = /obj/item/radio/headset/headset_srv
 	uniform = /obj/item/clothing/under/rank/civilian/janitor
 	backpack_contents = list(/obj/item/modular_computer/tablet/preset/advanced=1)
+	skillchips = list(/obj/item/skillchip/job/engineer, /obj/item/skillchip/light_remover)
```
Comment (Fikou) [inline code/modules/jobs/job_types/janitor.dm:27]: unmentioned balance change thats also really powerful 🤔

Comment (cacogen) [reply code/modules/jobs/job_types/janitor.dm:27]: One is removing light tubes without gloves while the other is door wires? Is that right?

Comment (Jared-Fogle) [reply code/modules/jobs/job_types/janitor.dm:27]: yes

Comment (coiax) [reply code/modules/jobs/job_types/janitor.dm:27]: Oh shit, I didn't mean to leave that in, that was for testing :sweat:

Comment (cacogen) [reply code/modules/jobs/job_types/janitor.dm:27]: Light remover makes sense. I don't get the engineer one. The janitor should have a chipset that makes them clean faster.

Comment (coiax) [reply code/modules/jobs/job_types/janitor.dm]: Feel free to make a PR adding the light remover skillchip to the janitor after this one is merged, but this is just supposed to be pure refactoring.

**Pull request merged**